nextflow.config

```diff
@@ -1,6 +1,15 @@
 process {
     executor = 'slurm'
     cache = 'lenient'
+    resourceLimits {
+        // These limits are the maximum across all partitions and QOS
+        // on the cluster. Thus, these are mostly taken from the bonus QOS, which
+        // has the most lenient resource limits.
+        // See `sacctmgr show QOS Format=name%15,MaxTRESPU%110,MaxWall`
+        cpus = 6896
```

Comment:
My understanding of the docs is that the resourceLimits should be used to constrain the process to the maximum resources available on a _single node_ not on the entire cluster. e.g. it doesn't make sense to submit a 100 CPU job, if the larges node only has 50CPUs. So cpu limits should be `cpus = 196` and `memory = "3011.GB"` to match the `cl-n01` node.
Can confirm that setting the resource limits will automatically lower the job resource request to match that limit.

Reply:
Yes that's correct, thanks.